test/rendering/attributes_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule RenderAttributesTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case#, async: true
```

Comment:
Oh crap sorry for this. I havn't review my commit as usual.
David Authier @davidawea <http://twitter.com/davidawea>

Le 16/03/2016 18:24, Louis Pilfold a écrit :
>
> In test/rendering/attributes_test.exs 
> <https://github.com/slime-lang/slime/pull/94#discussion_r56377399>:
>
> > @@ -1,5 +1,5 @@
> >  defmodule RenderAttributesTest do
> > -  use ExUnit.Case, async: true
> > +  use ExUnit.Case#, async: true
>
> Remove this commented out code please
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/slime-lang/slime/pull/94/files/6d623716fb1fe3ad87769d957a2da878315e3b15#r56377399>
>